sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/administration/DocumentModelAdministrationClient.java

```diff
@@ -68,6 +68,7 @@
      */
     public DocumentAnalysisClient getDocumentAnalysisClient() {
         return new DocumentAnalysisClientBuilder().endpoint(client.getEndpoint()).pipeline(client.getHttpPipeline())
+            .audience(client.getAudience())
```

Comment:
The `getAudience` method is used here in the sync client. Should this method just be package-private on the async client then?